db.ts

```diff
@@ -74,15 +74,15 @@ const createNewHighScore = async (user: User, typeOfQuiz: string, newHighScore:
 
 const deleteBlacklist = async (user: User, quoteId: string) => {
     try {
-        await client.db("TheOneQuiz").collection("Users").updateOne({ username: user.username }, { $pull: { blacklist: { quoteId: quoteId } } });
+        await client.db("TheOneQuiz").collection("Users").updateOne({ username: user.username }, { $pull: { blacklist: { quote_Id: quoteId } } });
```

Comment:
It has to be "quote_id", all lowercase. That's the only reason it doesn't work i think